README.developer.md

```diff
@@ -12,9 +12,9 @@
   - [Unit Tests](#unit-tests)
   - [Integration Tests](#integration-tests)
   - [Running](#running)
-    - [hirte](#hirte)
-    - [hirte-agent](#hirte-agent)
-    - [hirtectl](#hirtectl)
+    - [bluechi](#bluechi)
+    - [bluechi-agent](#bluechi-agent)
+    - [bluechi-ctl](#bluechi-ctl)
```

Comment:
Not sure we agreed on the '-' bluechictl vs bluechi-ctl.

Reply:
The @mwperina doc says bluechi-ctl just following the doc.

Reply:
Yes, the doc states `bluechi-ctl`. Not sure if this has been discussed in the sync, but I'd prefer `bluechictl` and use the naming pattern of `kubectl` and `systemctl`.

Reply:
I thought about the naming and `bluechi-ctl` seemed to easier to spell and more aligned with `bluechi-agent` or `bluechi-proxy` than `bluechictl`. As mentioned it's like that in the document and nobody complained so far.

Reply:
OK, changed to `bluechictl`